### PR TITLE
Clarify the fixup instructions for the precommit hook.

### PR DESCRIPTION
**Pull request by alanf**
Let the people know that there's a fast way to fix and stage changes!

screen shot 2015-10-27 at 12 25 48 pm

### CODE REVIEW

**Comment by alanf** [comment]
cc @orta

File: format-objc-hook [hunk 1]
```diff
@@ -19,18 +19,24 @@ objc_files=$(objc_files_to_format "$1")
...
-	difference=$("$DIR"/format-objc-file-dry-run.sh "$file" | diff "$file" - | wc -l)
+    difference=$("$DIR"/format-objc-file-dry-run.sh "$file" | diff "$file" - | wc -l)
 	
     if [ $difference -gt 0 ]; then
+        if [ $success -eq 0 ]; then
+            echo -e "🚸 Format and stage individual files:"
+        fi
     	# This is what the dev can run to fixup an individual file
-    	echo "\"$DIR\"/format-objc-file.sh '$file' && git add '$file';"
+    	echo -e "\"$DIR\"/format-objc-file.sh '$file' && git add '$file';"
```
Comment (justinseanmartin) [inline format-objc-hook]: NIT: I don't think you need the `-e` here, because there is no `\n` on the line.

File: format-objc-hook [hunk 2]
```diff
@@ -19,18 +19,24 @@ objc_files=$(objc_files_to_format "$1")
...
     	success=1
```
Comment (justinseanmartin) [inline format-objc-hook]: Not necessary, pre-existing issue - Having `success=1` mean that it is failing seems counter-intuitive to me, but I understand the exit code should be non-zero.

**Comment by kyleve** [comment]
LGTM

File: format-objc-hook (hunk 1 above)
Comment (justinseanmartin) [inline format-objc-hook]: With the below "fix all" message, it seems reasonable to remove the `git add` here, which would also address @orta's issue.

**Comment by justinseanmartin** [comment]
LGTM as well.

**Comment by orta** [comment]
Ace!

**Pull request merged**